Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/Coupled/Examples/package.mo

```diff
@@ -0,0 +1,9 @@
+within Buildings.Electrical.AC.ThreePhasesBalanced.Loads.MotorDrive.Coupled;
+package Examples "Collection of models that illustrate model use and test models"
+  extends Modelica.Icons.ExamplesPackage;
+
+
+annotation (Documentation(info="<html>
+<p>This package contains examples for the use of models that can be found in <a href=\"modelica://MotorDrive.Coupled\">MotorDrive.Coupled</a>.</p>
```

Comment:
better use the full path link

Reply:
Thank you for catching this, I have corrected the path to the full path in all relevant files.